lib/ood_core/cluster.rb

```diff
@@ -147,6 +147,15 @@ def allow?
       @allow = acls.all?(&:allow?)
     end
 
+    # Whether this cluster supports SSH to batch connect nodes
+    # @return [Boolean] whether cluster supports SSH to batch connect node
+    def batch_connect_ssh_allow?(default = true)
+      return @batch_connect_ssh_allow if defined?(@batch_connect_ssh_allow)
+      return @batch_connect_ssh_allow = default if batch_connect_config.nil?
+
+      @batch_connect_ssh_allow = batch_connect_config.fetch(:ssh_allow, default)
```

Comment:
I don't think we need default in the API. 

 `@batch_connect_config.fetch(:ssh_allow, true)` should be just fine. or even `@batch_connect_config&.fetch(:ssh_allow, true) || true` just to be extra double safe (though it looks like batch_connect_config should always be at least an empty {})

Reply:
I fixed this, I think this was left over from an idea where the default would be pulled from the global env config, but I think chaining them all makes more sense.